### PR TITLE
Added transactions mapping and staff comment

### PR DESCRIPTION
**Pull request by cameron-freshworks**
*Issue #:* bcgov/entity#8248

*Description of changes:*
* Included mapping in transaction history list for split request types.
* Applied staff comments to transactions history.

*Notes:*
* The api side of this additional `comment` property is still in code review for Namex.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the namex license (Apache 2.0).

### CODE REVIEW

File: client/src/components/application/Transactions.vue [hunk 1]
```diff
@@ -96,7 +96,7 @@
             v-for="(transaction, index) in filteredTransactions"
             :key="index"
             class="transaction-item"
-          >
+          >{{transaction.comment = 'Testing testing 123 Testing testing 123 Testing testing 123 Testing testing 123 Testing testing 123 '}}
```
Comment (severinbeauvais) [inline client/src/components/application/Transactions.vue:99]: To be removed?

File: client/src/components/application/Transactions.vue [hunk 2]
```diff
@@ -164,10 +178,15 @@
...
 
 <script>
+/* eslint-disable */
   import { mapState } from 'vuex'
   import moment from 'moment'
   import CompNameIcon from './Examine/CompNameIcon'
   import Spinner from './spinner'
+  import RequestActionCode from '@/enums/request-action-code'
+  import RequestTypeCode from '@/enums/request-type-code'
+  import EntityTypeCode from '@/enums/entity-type-code'
```
Comment (severinbeauvais) [inline client/src/components/application/Transactions.vue:188]: How about creating an index file and importing all 3 on the same line?

File: client/src/components/application/Transactions.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/application/Transactions.vue:99]: haha yea, thank you!

File: client/src/components/application/Transactions.vue (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/components/application/Transactions.vue:188]: I did try this, but wasn't working quite the same as we we're used too. 
Can see if there is way tho!

File: client/src/components/application/Transactions.vue (hunk 1 above)
Comment (cameron-freshworks) [reply client/src/components/application/Transactions.vue:99]: Removed in next commit 👍

**Review by severinbeauvais — APPROVED** [review]
Looks fine. Do what you need to.

File: client/src/components/application/Transactions.vue (hunk 2 above)
Comment (cameron-freshworks) [reply client/src/components/application/Transactions.vue:188]: Yea, i've tried a few things unsuccessfully. Will continue looking however.

Comment (severinbeauvais) [reply client/src/components/application/Transactions.vue:188]: Don't worry too much about it.

**Pull request merged**